### PR TITLE
Touch screens adaptative functionality.

### PR DESCRIPTION
**Pull request by juliocri**

### CODE REVIEW

File: src/board-explorer-app/board-explorer-app.html [hunk 1]
```diff
@@ -968,7 +972,7 @@
       selected.forEach(function(refdes) {
         /* Disable the button that matches this refdes */
         Array.prototype.forEach.call(this.$.layout.querySelectorAll("paper-button[match='"+ refdes +"']"), function(el) {
-          el.disabled = true;
+          el.disabled = false;
         });
```
Comment (jketreno) [inline src/board-explorer-app/board-explorer-app.html]: Is this change intentional? If so, the comment should be changed to indicate that the code is enabling the button that matches the refdes.

Comment (juliocri) [reply src/board-explorer-app/board-explorer-app.html]: Yeap was intentionally, the coment was updated in the last commit.
